Unable to get IP address with wireless network interface

Unable to get IP address through Wireless network interface using DHCP. 

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Network menu item.
2. Select wireless interface
3. Select Enable for WAN and apply.
4. Configure settings in wireless tab and apply.
5. Go to TCP/IP and refresh or press Renew DHCP Lease button.

**Expected behavior**
Network interface get proper ip address, gateway and subnet mask.


**Target Environment:**
 - Intel Up2 
 - Linux intel-up2 5.4.0-73-generic #82~18.04.1-Ubuntu SMP Fri Apr 16 15:10:02 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
 - KURA_5.0.0-SNAPSHOT

```2021-06-15T10:10:26,681 [qtp571101017-4635] INFO  o.e.k.n.a.NetworkAdminServiceImpl - getWifiHotspots() :: scanning for available access points ...
2021-06-15T10:10:26,687 [qtp571101017-4635] INFO  o.e.k.l.n.u.IwScanTool - scan() :: executing: iw dev wlp4s0 scan
2021-06-15T10:10:26,691 [qtp571101017-4635] INFO  o.e.k.l.n.u.IwScanTool - scan() :: iw dev wlp4s0 scan command returns status = 161
2021-06-15T10:10:26,692 [qtp571101017-4635] ERROR o.e.k.l.n.u.IwScanTool - scan() :: failed to execute iw dev wlp4s0 scan error code is 161
2021-06-15T10:10:26,692 [qtp571101017-4635] ERROR o.e.k.l.n.u.IwScanTool - scan() :: STDERR: command failed: Operation not supported (-95)

2021-06-15T10:10:26,692 [qtp571101017-4635] ERROR o.e.k.w.s.GwtNetworkServiceImpl - Failed
org.eclipse.kura.KuraException: An internal error occurred. scan operation has failed
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiHotspotList(NetworkAdminServiceImpl.java:1230)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.findWifiHotspots(GwtNetworkServiceImpl.java:1014)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.findWifiHotspots(GwtNetworkServiceImpl.java:1)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.processCall(OsgiRemoteServiceServlet.java:208)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.service(OsgiRemoteServiceServlet.java:95)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.servlet.internal.HttpServiceRuntimeImpl$LegacyServlet.service(HttpServiceRuntimeImpl.java:1459)
        at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:159)
        at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:67)
        at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:117)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.dispatch(ProxyServlet.java:147)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.preprocess(ProxyServlet.java:115)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.service(ProxyServlet.java:104)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.jetty.internal.HttpServerManager$InternalHttpServiceServlet.service(HttpServerManager.java:305)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:713)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:540)
        at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:395)
        at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:161)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:383)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:882)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1036)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.kura.KuraException: Unable to execute system process
        at org.eclipse.kura.linux.net.util.IwScanTool.scan(IwScanTool.java:101)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiAccessPoints(NetworkAdminServiceImpl.java:217)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiHotspotList(NetworkAdminServiceImpl.java:1203)
        ... 60 more
2021-06-15T10:10:26,697 [qtp571101017-4635] WARN  o.e.k.w.s.u.KuraExceptionHandler - RPC service non-application error
org.eclipse.kura.KuraException: An internal error occurred. scan operation has failed
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiHotspotList(NetworkAdminServiceImpl.java:1230)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.findWifiHotspots(GwtNetworkServiceImpl.java:1014)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.findWifiHotspots(GwtNetworkServiceImpl.java:1)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.processCall(OsgiRemoteServiceServlet.java:208)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.service(OsgiRemoteServiceServlet.java:95)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.servlet.internal.HttpServiceRuntimeImpl$LegacyServlet.service(HttpServiceRuntimeImpl.java:1459)
        at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:159)
        at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:67)
        at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:117)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.dispatch(ProxyServlet.java:147)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.preprocess(ProxyServlet.java:115)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.service(ProxyServlet.java:104)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.jetty.internal.HttpServerManager$InternalHttpServiceServlet.service(HttpServerManager.java:305)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:713)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:540)
        at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:395)
        at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:161)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:383)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:882)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1036)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.kura.KuraException: Unable to execute system process
        at org.eclipse.kura.linux.net.util.IwScanTool.scan(IwScanTool.java:101)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiAccessPoints(NetworkAdminServiceImpl.java:217)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.getWifiHotspotList(NetworkAdminServiceImpl.java:1203)
        ... 60 more```

After a reboot:
```2021-06-15T10:23:17,860 [WifiMonitor Thread] INFO  o.e.k.l.n.u.IwScanTool - scan() :: iw dev wlp4s0 scan command returns status = 0
2021-06-15T10:23:17,861 [WifiMonitor Thread] INFO  o.e.k.l.n.u.IwScanTool - scan() :: the 'iw scan' command executed successfully, parsing output ...
2021-06-15T10:23:17,863 [WifiMonitor Thread] INFO  o.e.k.n.a.m.WifiMonitorServiceImpl - monitor() :: found access point - enable wlp4s0 in infra mode
2021-06-15T10:23:18,007 [WifiMonitor Thread] WARN  o.e.k.n.a.m.InterfaceStateBuilder - setWifiLinkState() :: !! Link is down for the wlp4s0 interface. isSupplicantRunning? false isIfaceInManagedMode? true
2021-06-15T10:23:18,014 [WifiMonitor Thread] INFO  o.e.k.n.a.NetworkAdminServiceImpl - bringing interface wlp4s0 up
2021-06-15T10:23:18,073 [WifiMonitor Thread] WARN  o.e.k.n.a.NetworkAdminServiceImpl - enableWifiInterface() :: Starting wpa_supplicant ... driver=nl80211
2021-06-15T10:23:18,094 [WifiMonitor Thread] INFO  o.e.k.l.n.w.WpaSupplicantManager - starting wpa_supplicant for the wlp4s0 interface -> [wpa_supplicant, -B, -D, nl80211, -i, wlp4s0, -c, /etc/wpa_supplicant-wlp4s0.conf]
2021-06-15T10:23:21,444 [AbstractNtpClockSyncProvider:schedule] INFO  o.e.k.l.c.AbstractNtpClockSyncProvider - Try to sync clock (21)
2021-06-15T10:23:21,446 [AbstractNtpClockSyncProvider:schedule] WARN  o.e.k.l.c.JavaNtpClockSyncProvider - Error while synchronizing System Clock with NTP host 0.pool.ntp.org. Please verify network connectivity ...
2021-06-15T10:23:26,444 [AbstractNtpClockSyncProvider:schedule] INFO  o.e.k.l.c.AbstractNtpClockSyncProvider - Try to sync clock (22)
2021-06-15T10:23:26,445 [AbstractNtpClockSyncProvider:schedule] WARN  o.e.k.l.c.JavaNtpClockSyncProvider - Error while synchronizing System Clock with NTP host 0.pool.ntp.org. Please verify network connectivity ...
2021-06-15T10:23:31,444 [AbstractNtpClockSyncProvider:schedule] INFO  o.e.k.l.c.AbstractNtpClockSyncProvider - Try to sync clock (23)
2021-06-15T10:23:31,445 [AbstractNtpClockSyncProvider:schedule] WARN  o.e.k.l.c.JavaNtpClockSyncProvider - Error while synchronizing System Clock with NTP host 0.pool.ntp.org. Please verify network connectivity ...
2021-06-15T10:23:32,515 [qtp1368194579-87] ERROR o.e.k.n.a.v.l.HostapdConfigWriter - Failed to generate hostapd configuration file for wlp4s0 interface
2021-06-15T10:23:32,516 [qtp1368194579-87] ERROR o.e.k.n.a.NetworkConfigurationServiceImpl - Error updating the configuration
org.eclipse.kura.KuraException: An internal error occurred.
        at org.eclipse.kura.KuraException.internalError(KuraException.java:123)
        at org.eclipse.kura.net.admin.visitor.linux.HostapdConfigWriter.writeConfig(HostapdConfigWriter.java:124)
        at org.eclipse.kura.net.admin.visitor.linux.HostapdConfigWriter.visit(HostapdConfigWriter.java:93)
        at org.eclipse.kura.net.admin.visitor.linux.WifiConfigWriter.visit(WifiConfigWriter.java:72)
        at org.eclipse.kura.net.admin.visitor.linux.LinuxWriteVisitor.visit(LinuxWriteVisitor.java:51)
        at org.eclipse.kura.core.net.NetworkConfiguration.accept(NetworkConfiguration.java:159)
        at org.eclipse.kura.net.admin.NetworkConfigurationServiceImpl.updated(NetworkConfigurationServiceImpl.java:253)
        at org.eclipse.kura.net.admin.NetworkConfigurationServiceImpl.setNetworkConfiguration(NetworkConfigurationServiceImpl.java:217)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.submitNetworkConfiguration(NetworkAdminServiceImpl.java:1391)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.updateWifiInterfaceConfig(NetworkAdminServiceImpl.java:752)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.updateNetInterfaceConfigurations(GwtNetworkServiceImpl.java:867)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.processCall(OsgiRemoteServiceServlet.java:208)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.service(OsgiRemoteServiceServlet.java:95)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.servlet.internal.HttpServiceRuntimeImpl$LegacyServlet.service(HttpServiceRuntimeImpl.java:1459)
        at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:159)
        at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:67)
        at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:117)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.dispatch(ProxyServlet.java:147)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.preprocess(ProxyServlet.java:115)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.service(ProxyServlet.java:104)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.jetty.internal.HttpServerManager$InternalHttpServiceServlet.service(HttpServerManager.java:305)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:713)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:540)
        at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:395)
        at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:161)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:383)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:882)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1036)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.kura.KuraException: An internal error occurred.
        at org.eclipse.kura.KuraException.internalError(KuraException.java:123)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.copyFile(WriterHelper.java:45)
        at org.eclipse.kura.net.admin.visitor.linux.HostapdConfigWriter.generateHostapdConf(HostapdConfigWriter.java:204)
        at org.eclipse.kura.net.admin.visitor.linux.HostapdConfigWriter.writeConfig(HostapdConfigWriter.java:121)
        ... 67 more
Caused by: java.nio.file.FileSystemException: /etc/hostapd.conf.tmp: Operation not permitted
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:91)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileAttributeViews$Posix.setMode(UnixFileAttributeViews.java:238)
        at sun.nio.fs.UnixFileAttributeViews$Posix.setPermissions(UnixFileAttributeViews.java:260)
        at java.nio.file.Files.setPosixFilePermissions(Files.java:2045)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.setPermissions(WriterHelper.java:53)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.copyFile(WriterHelper.java:43)
        ... 69 more```
        
        ```2021-06-15T10:24:28,856 [WifiMonitor Thread] INFO  o.e.k.l.n.u.IwScanTool - scan() :: executing: iw dev wlp4s0 scan
2021-06-15T10:24:31,444 [AbstractNtpClockSyncProvider:schedule] INFO  o.e.k.l.c.AbstractNtpClockSyncProvider - Try to sync clock (35)
2021-06-15T10:24:31,444 [AbstractNtpClockSyncProvider:schedule] WARN  o.e.k.l.c.JavaNtpClockSyncProvider - Error while synchronizing System Clock with NTP host 0.pool.ntp.org. Please verify network connectivity ...
2021-06-15T10:24:31,987 [WifiMonitor Thread] INFO  o.e.k.l.n.u.IwScanTool - scan() :: iw dev wlp4s0 scan command returns status = 0
2021-06-15T10:24:31,988 [WifiMonitor Thread] INFO  o.e.k.l.n.u.IwScanTool - scan() :: the 'iw scan' command executed successfully, parsing output ...
2021-06-15T10:24:31,993 [WifiMonitor Thread] INFO  o.e.k.n.a.m.WifiMonitorServiceImpl - monitor() :: found access point - enable wlp4s0 in infra mode
2021-06-15T10:24:32,115 [WifiMonitor Thread] INFO  o.e.k.n.a.NetworkAdminServiceImpl - bringing interface wlp4s0 up
2021-06-15T10:24:32,171 [WifiMonitor Thread] INFO  o.e.k.c.i.l.e.ExecutorUtil - Attempting to send SIGKILL to process with pid 6256
2021-06-15T10:24:32,185 [WifiMonitor Thread] WARN  o.e.k.n.a.NetworkAdminServiceImpl - enableWifiInterface() :: Starting wpa_supplicant ... driver=nl80211
2021-06-15T10:24:32,206 [WifiMonitor Thread] INFO  o.e.k.l.n.w.WpaSupplicantManager - starting wpa_supplicant for the wlp4s0 interface -> [wpa_supplicant, -B, -D, nl80211, -i, wlp4s0, -c, /etc/wpa_supplicant-wlp4s0.conf]
2021-06-15T10:24:34,528 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - enable() :: disabling DHCP client for wlp4s0
2021-06-15T10:24:34,554 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - disable() :: killing DHCP client for wlp4s0
2021-06-15T10:24:34,570 [WifiMonitor Thread] INFO  o.e.k.c.i.l.e.ExecutorUtil - Attempting to send SIGKILL to process with pid 6967
2021-06-15T10:24:34,574 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - enable() :: Starting DHCP client for wlp4s0
2021-06-15T10:24:36,444 [AbstractNtpClockSyncProvider:schedule] INFO  o.e.k.l.c.AbstractNtpClockSyncProvider - Try to sync clock (36)
2021-06-15T10:24:43,832 [DnsMonitorServiceImpl] INFO  o.e.k.n.a.m.DnsMonitorServiceImpl - Change to DNS - setting dns servers: [172.16.1.1]
2021-06-15T10:24:43,833 [DnsMonitorServiceImpl] ERROR o.e.k.l.n.d.LinuxDns - writeDnsFile() :: failed to write the /etc/resolv.conf file
java.io.FileNotFoundException: /etc/resolv.conf (Operation not permitted)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at org.eclipse.kura.linux.net.dns.LinuxDns.writeDnsFile(LinuxDns.java:336)
        at org.eclipse.kura.linux.net.dns.LinuxDns.setDnServers(LinuxDns.java:94)
        at org.eclipse.kura.net.admin.monitor.DnsMonitorServiceImpl.setDnsServers(DnsMonitorServiceImpl.java:323)
        at org.eclipse.kura.net.admin.monitor.DnsMonitorServiceImpl.lambda$0(DnsMonitorServiceImpl.java:126)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
2021-06-15T10:24:43,834 [DnsMonitorServiceImpl] INFO  o.e.k.n.a.m.DnsMonitorServiceImpl - Detected DNS resolv.conf change - restarting DNS proxy
2021-06-15T10:24:44,701 [EventAdmin Async Event Dispatcher Thread] INFO  o.e.k.n.a.m.DnsMonitorServiceImpl - Change to DNS - setting dns servers: [172.16.1.1]
2021-06-15T10:24:44,702 [EventAdmin Async Event Dispatcher Thread] ERROR o.e.k.l.n.d.LinuxDns - writeDnsFile() :: failed to write the /etc/resolv.conf file
java.io.FileNotFoundException: /etc/resolv.conf (Operation not permitted)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at org.eclipse.kura.linux.net.dns.LinuxDns.writeDnsFile(LinuxDns.java:336)
        at org.eclipse.kura.linux.net.dns.LinuxDns.setDnServers(LinuxDns.java:94)
        at org.eclipse.kura.net.admin.monitor.DnsMonitorServiceImpl.setDnsServers(DnsMonitorServiceImpl.java:323)
        at org.eclipse.kura.net.admin.monitor.DnsMonitorServiceImpl.updateDnsResolverConfig(DnsMonitorServiceImpl.java:250)
        at org.eclipse.kura.net.admin.monitor.DnsMonitorServiceImpl.handleEvent(DnsMonitorServiceImpl.java:243)
        at org.eclipse.equinox.internal.event.EventHandlerWrapper.handleEvent(EventHandlerWrapper.java:205)
        at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:203)
        at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:1)
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
2021-06-15T10:24:44,899 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - enable() :: disabling DHCP client for wlp4s0
2021-06-15T10:24:44,919 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - disable() :: killing DHCP client for wlp4s0
2021-06-15T10:24:44,934 [WifiMonitor Thread] INFO  o.e.k.c.i.l.e.ExecutorUtil - Attempting to send SIGKILL to process with pid 7271
2021-06-15T10:24:44,938 [WifiMonitor Thread] INFO  o.e.k.l.n.d.DhcpClientManager - enable() :: Starting DHCP client for wlp4s0```

I can confirm the behaviour.
The problem is on all `.conf` files previously created with root as owner by KURA_4.1.2 .


```2021-06-17T11:58:44,687 [qtp1841268744-218677] ERROR o.e.k.n.a.v.l.WpaSupplicantConfigWriter - Failed to configure WPA Supplicant
2021-06-17T11:58:44,689 [qtp1841268744-218677] ERROR o.e.k.n.a.NetworkConfigurationServiceImpl - Error updating the configuration
org.eclipse.kura.KuraException: An internal error occurred.
        at org.eclipse.kura.KuraException.internalError(KuraException.java:123)
        at org.eclipse.kura.net.admin.visitor.linux.WpaSupplicantConfigWriter.writeAndMoveFile(WpaSupplicantConfigWriter.java:233)
        at org.eclipse.kura.net.admin.visitor.linux.WpaSupplicantConfigWriter.writeConfig(WpaSupplicantConfigWriter.java:171)
        at org.eclipse.kura.net.admin.visitor.linux.WpaSupplicantConfigWriter.visit(WpaSupplicantConfigWriter.java:87)
        at org.eclipse.kura.net.admin.visitor.linux.WifiConfigWriter.visit(WifiConfigWriter.java:72)
        at org.eclipse.kura.net.admin.visitor.linux.LinuxWriteVisitor.visit(LinuxWriteVisitor.java:51)
        at org.eclipse.kura.core.net.NetworkConfiguration.accept(NetworkConfiguration.java:159)
        at org.eclipse.kura.net.admin.NetworkConfigurationServiceImpl.updated(NetworkConfigurationServiceImpl.java:253)
        at org.eclipse.kura.net.admin.NetworkConfigurationServiceImpl.setNetworkConfiguration(NetworkConfigurationServiceImpl.java:217)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.submitNetworkConfiguration(NetworkAdminServiceImpl.java:1391)
        at org.eclipse.kura.net.admin.NetworkAdminServiceImpl.updateWifiInterfaceConfig(NetworkAdminServiceImpl.java:752)
        at org.eclipse.kura.web.server.GwtNetworkServiceImpl.updateNetInterfaceConfigurations(GwtNetworkServiceImpl.java:867)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.processCall(OsgiRemoteServiceServlet.java:208)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
        at org.eclipse.kura.web.server.OsgiRemoteServiceServlet.service(OsgiRemoteServiceServlet.java:95)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.servlet.internal.HttpServiceRuntimeImpl$LegacyServlet.service(HttpServiceRuntimeImpl.java:1459)
        at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:159)
        at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:67)
        at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:117)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.dispatch(ProxyServlet.java:147)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.preprocess(ProxyServlet.java:115)
        at org.eclipse.equinox.http.servlet.internal.servlet.ProxyServlet.service(ProxyServlet.java:104)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.eclipse.equinox.http.jetty.internal.HttpServerManager$InternalHttpServiceServlet.service(HttpServerManager.java:305)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:713)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:540)
        at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:395)
        at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:161)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:383)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:882)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1036)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.kura.KuraException: An internal error occurred.
        at org.eclipse.kura.KuraException.internalError(KuraException.java:123)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.copyFile(WriterHelper.java:45)
        at org.eclipse.kura.net.admin.visitor.linux.WpaSupplicantConfigWriter.generateWpaSupplicantConf(WpaSupplicantConfigWriter.java:291)
        at org.eclipse.kura.net.admin.visitor.linux.WpaSupplicantConfigWriter.writeAndMoveFile(WpaSupplicantConfigWriter.java:228)
        ... 68 more
Caused by: java.nio.file.FileSystemException: /etc/wpa_supplicant.conf.tmp: Operation not permitted
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:91)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileAttributeViews$Posix.setMode(UnixFileAttributeViews.java:238)
        at sun.nio.fs.UnixFileAttributeViews$Posix.setPermissions(UnixFileAttributeViews.java:260)
        at java.nio.file.Files.setPosixFilePermissions(Files.java:2045)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.setPermissions(WriterHelper.java:53)
        at org.eclipse.kura.net.admin.visitor.linux.WriterHelper.copyFile(WriterHelper.java:43)
        ... 70 more```

--- a/kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh
+++ b/kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh
@@ -103,6 +103,13 @@ if [ ! -d /etc/logrotate.d/ ]; then
 fi
 cp ${INSTALL_DIR}/kura/install/kura.logrotate /etc/logrotate.d/kura
 
+#assigning possible .conf files ownership to kurad
+PATTERN="/etc/dhcpd*.conf* /etc/resolv.conf* /etc/wpa_supplicant*.conf* /etc/hostapd*.conf*"
+for FILE in $(ls $PATTERN 2>/dev/null)
+do
+  chown kurad:kurad $FILE
+done
+
 # Setup tmpfiles.d
 cp ${INSTALL_DIR}/kura/install/kura-tmpfiles.conf /usr/lib/tmpfiles.d/kura.conf
 

--- a/kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh
+++ b/kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh
@@ -96,6 +96,13 @@ fi
 cp ${INSTALL_DIR}/kura/install/logrotate.conf /etc/logrotate.conf
 cp ${INSTALL_DIR}/kura/install/kura.logrotate /etc/logrotate.d/kura
 
+#assigning possible .conf files ownership to kurad
+PATTERN="/etc/dhcpd*.conf* /etc/resolv.conf* /etc/wpa_supplicant*.conf* /etc/hostapd*.conf*"
+for FILE in $(ls $PATTERN 2>/dev/null)
+do
+  chown kurad:kurad $FILE
+done
+
 # execute patch_sysctl.sh from installer install folder
 chmod 700 ${INSTALL_DIR}/kura/install/patch_sysctl.sh
 ${INSTALL_DIR}/kura/install/patch_sysctl.sh ${INSTALL_DIR}/kura/install/sysctl.kura.conf /etc/sysctl.conf

--- a/kura/distrib/src/main/resources/raspberry-pi-2/kura_install.sh
+++ b/kura/distrib/src/main/resources/raspberry-pi-2/kura_install.sh
@@ -99,6 +99,13 @@ fi
 cp ${INSTALL_DIR}/kura/install/logrotate.conf /etc/logrotate.conf
 cp ${INSTALL_DIR}/kura/install/kura.logrotate /etc/logrotate.d/kura
 
+#assigning possible .conf files ownership to kurad
+PATTERN="/etc/dhcpd*.conf* /etc/resolv.conf* /etc/wpa_supplicant*.conf* /etc/hostapd*.conf*"
+for FILE in $(ls $PATTERN 2>/dev/null)
+do
+  chown kurad:kurad $FILE
+done
+
 # execute patch_sysctl.sh from installer install folder
 chmod 700 ${INSTALL_DIR}/kura/install/patch_sysctl.sh
 ${INSTALL_DIR}/kura/install/patch_sysctl.sh ${INSTALL_DIR}/kura/install/sysctl.kura.conf /etc/sysctl.conf

